source/docs/yearly-overview/returning-quickstart.rst

```diff
@@ -0,0 +1,15 @@
+
+Quick Start for Returning Teams
+===============================
+
+This section serves as a launching point for veteran team that needs to update to the current year's SW.
```

Comment:
Maybe say "veteran teams that need to update"  instead to improve sentence structure